src/ontogpt/cli.py

```diff
@@ -280,6 +280,26 @@ def pubmed_extract(pmid, template, output, output_format, **kwargs):
     write_extraction(results, output, output_format)
 
 
+@main.command()
```

Comment:
note we already have `pubmed-extract` which I think does the same thing...?

Reply:
or rather `search-and-extract`

Reply:
It may be more intuitive to modify `pubmed-extract` to accept more than one PMID

Reply:
though that also assumes you already have the PMIDs, so maybe more like providing a flag to do the search, output the PMID list, then pass that to extract

While making the distinction that there's also a use case for search -> get PMIDs -> extract keywords -> new search -> new PMIDs -> extract/summarize

Reply:
my $0.02 I'd say why not make a command to do this in one go, rather than making the user run two commands (one to get PMIDs and another to extract info)